framework/validators/UniqueValidator.php

```diff
@@ -121,7 +121,7 @@ public function validateAttribute($model, $attribute)
         $targetClass = $this->getTargetClass($model);
         $targetAttribute = $this->targetAttribute === null ? $attribute : $this->targetAttribute;
         $rawConditions = $this->prepareConditions($targetAttribute, $model, $attribute);
-        $conditions[] = $this->targetAttributeJunction === 'or' ? 'or' : 'and';
+        $conditions = [ $this->targetAttributeJunction === 'or' ? 'or' : 'and' ] ;
```

Comment:
Why?

Reply:
Just want to fix [scrunitizer issue](https://scrutinizer-ci.com/g/yiisoft/yii2/issues/master/files/framework/validators/UniqueValidator.php?selectedLabels%5B0%5D=3&selectedLabels%5B1%5D=10&orderField=path&order=asc&honorSelectedPaths=0)

Reply:
OK.

Reply:
Extra spaces removed